fredericia/web
# FKDK-cleanup Moving contrib and common custom modules to profile http…

In scope of prepraring os2web profile for new project we are going to cleanup sites/all folder and move all common functionality and code to profile

See https://github.com/bellcom/os2web_profile/pull/6

How to test:
1. go to /profiles/osweb_profile folder fetch and checkout to `FKDK-cleanup` branch
2. go to /sites/fredericia.dk folder fetch and checkout to `FKDK-cleanup` branch
3. call `drush rr`
3.1 If you will got a bug related to composer manager try to fix it woth direct SAL request
```
drush sqlq "UPDATE system SET filename = 'profiles/os2web/modules/contrib/composer_manager/composer_manager.module' WHERE name = 'composer_manager' AND type = 'module';"
```
5. check the project
